modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -9,6 +9,30 @@ With an Enterprise license, you can additionally enable _Continuous Data Balanci
 
 Cluster balancing protects you from unbalanced systems that saturate resources on one or more brokers. This can affect throughput and latency. Furthermore, a cluster with replicas on a down broker risks availability loss if more brokers fail, and a cluster that keeps losing brokers without healing eventually risks data loss.
 
+The following table summarizes the various balancing options in Redpanda:
+
+
+.Cluster balancing options
+[cols="3,1,6"]
+|===
+|Balancer |License |Description
+
+|Leadership balancing
```

Comment:
Can we unify the balancer names used here in the table with what's later in the topic?

So, Leadership Balancing -> Partition Leadership Balancing